packages/pinia/src/storeToRefs.ts

```diff
@@ -68,9 +68,11 @@ type _ToStateRefs<SS> =
  * Extracts the return type for `storeToRefs`.
  * Will convert any `getters` into `ComputedRef`.
  */
-export type StoreToRefs<SS extends StoreGeneric> = _ToStateRefs<SS> &
-  ToRefs<PiniaCustomStateProperties<StoreState<SS>>> &
-  _ToComputedRefs<StoreGetters<SS>>
+export type StoreToRefs<SS extends StoreGeneric> = SS extends unknown
```

Comment:
Why does this fix the problem? Isn't `x extends unknown` always true? I want to add a note comment for my future self and other contributors
TS is sometimes so weird 🫠

Reply:
When conditional types act on a generic type, they become distributive when given a union type ([Distributive Conditional Types](https://www.typescriptlang.org/docs/handbook/2/conditional-types.html#distributive-conditional-types)).

In this case, it will always resolve to the first branch because `unknown` is the top type in TypeScript.

Reply:
Thanks! This makes me wonder if there isn't another bug up the chain of types. But this is already good enough!